en/editor/project/index.md

```diff
@@ -81,6 +169,66 @@ If you want to replace all the preset options currently used for image asset, yo
 
 ![Layers-camera](./index/layers-camera.png)
 
+## Physics
+
+![Physics](./index/physics-index.png)
+
+This physics configuration will be in effect only when the **Physics Module** is enabled in the **Engine Module**.
+
+![Physics-In-Engine](./index/physics-in-engine.png)
+
+The physical configuration will be used in the project when preview and release. The physical effects are reflected in the control of *gravity, friction, kinetic energy transfer, and collision detection*.
+
+### Property description
+
+- `gravity` Gravity vector, positive and negative values ​​reflect the directionality. **Default:** `{ x: 0, y: -10, z: 0 }`.
```

Comment:
Gravity direction vector, the sign means the positive or negative direction on the axis